### ISSUE TITLE
Group to Group Communication

### ISSUE THREAD

**Issue by King-Mob** [Wobbly-App/techno-syndicalism#16]
A messaging system for groups to groups

**Comment by marcaaron** [comment]
Curious about what is meant by this. A member of one group can communicate with a member from another group? Or more like a moderator of one group can coordinate with another moderator. Is it correct to assuming that messaging will mainly take place within a group via posts and comments, but that users from a group should also be exposed to the users within the same group for direct messaging?

**Comment by King-Mob** [comment]
It's a new thing that hasn't got a design pattern for it yet. A real world analogue of it would be delegates from one group meeting delegates from another one. But that's more like, the function we're trying to fill rather than the system we have to copy or model this feature on.

This is definitely not an MVP feature, it takes secure, vibrant local nodes as a prerequisite, and then asks, how do these nodes talk to each other? And I'm pretty certain an elegant solution that "feels right" is only going to come through trial and improvement.

But here's an inelegant starting point: Node1 has a button somewhere on its home screen called outward comms. Any member can press that, they are taken to a screen with a list of all the other nodes. They select Node2.
Node 2 gets an automated post saying, New Node-to-Node Communication, to let its members know about it.
Now, each Node has a new screen available, where members of Node1 and Node2 can chat freely, where their usernames maybe are highlighted in different colours to show where they're talking from.
So basically a big IM with a bit of a clue on identities.

That's the idea to do better than!
